### ISSUE TITLE
¿Podrían poner la página con ejercicios que recomendó el profe la semana pasada?

### ISSUE THREAD

**Issue by pafuenzalidaw** [IIC2115/Syllabus#43]
Especifica acá el detalle de tu problema, recuerda que el título debe ser autoexplicativo.
**Quejas y reclamos vía mail a `iic2115@ing.puc.cl`.**

**Comment by pcseisdedos** [comment]
Claro, es este  https://blog.usejournal.com/500-data-structures-and-algorithms-practice-problems-35afe8a1e222

Saludos,
Pablo 😁👍
